<ui version="4.0" >
 <class>MailingLabels</class>
 <widget class="QWidget" name="MailingLabels" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mailing Labels</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources/MIM.qrc" >:/icons/MIM.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" >
     <widget class="QPushButton" name="mailMergeButton" >
      <property name="text" >
       <string>Mail Merge...</string>
      </property>
      <property name="icon" >
       <iconset resource="resources/MIM.qrc" >:/icons/page_white_word.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QPushButton" name="printButton" >
      <property name="text" >
       <string>Print...</string>
      </property>
      <property name="icon" >
       <iconset resource="resources/MIM.qrc" >:/icons/printer.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QComboBox" name="categorySelect" >
      <property name="toolTip" >
       <string>Select a category</string>
      </property>
      <property name="statusTip" >
       <string>Select a category</string>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/MIM.qrc" />
 </resources>
 <connections/>
</ui>
